Library/Homebrew/development_tools.rb

```diff
@@ -100,9 +100,10 @@ def subversion_handles_most_https_certificates?
 
     def build_system_info
       {
-        "os"         => ENV["HOMEBREW_SYSTEM"],
-        "os_version" => OS_VERSION,
-        "cpu_family" => Hardware::CPU.family,
+        "os"                => ENV["HOMEBREW_SYSTEM"],
+        "os_version"        => OS_VERSION,
+        "cpu_family"        => Hardware::CPU.family,
+        "oldest_cpu_family" => Hardware.oldest_cpu,
```

Comment:
I'd like this to be only stored on Linux. There's no need for it to be in the tab on macOS.

Reply:
See https://github.com/Homebrew/brew/pull/10982#discussion_r606389221